api/db/migrations/20211126144130_add-disabled-at-column-in-certification-center-memberships-table.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'certification-center-memberships';
```

Comment:
@Anne-Gaelle-S comment on ajoute une nouvelle migration ? Via une commande dans le terminal ?

Reply:
un `npm run db:migrate` va ajouter cette migration à ta table en local.

Pour créer le fichier de migration sinon, on fait un `npm run db:new-migration <nom-du-fichier>`

Reply:
Oui, je l'ai noté dans la partie # Pour tester de la description de la PR :)

Reply:
Ou à moins que c'est vraiment la création du fichier qui t'intéresse ? Dans ce cas c'est `db:new-migration` de mémoire (tu trouveras tous les scripts dans le `package.json` de l'api sinon ^^

Reply:
Oui c'était plutôt la création du fichier merci, sur d'autres projets on avait l'ORM qui venait avec sa cli. Je regarde le package.json et scripts associés, merci !